### ISSUE TITLE
Facility String Compile Error

### ISSUE THREAD

**Issue by gregm711** [packethost/docker-machine-driver-packet#32]
I was following the docs to install the driver and ran into the following error when running make:

```
./driver.go:193:3: cannot use d.Facility (type string) as type []string in field value
```

I saw that (https://github.com/packethost/docker-machine-driver-packet/pull/31) should fix this, but some of the changes were up for discussion. As server location isn't a huge deal for my project I removed the facility parameter so I could carry on. 

This is from a clean install of the latest version.

**Comment by benjamb** [comment]
@gregm711 You can also make sure the packngo version is at 4099eefca4977c9a4909da715fa11055cdcf43c6 or earlier.

**Comment by benjamb** [comment]
I agree, breaking API isn't great. There was mention in the same MR (#31) of locking dependencies down, which could certainly help in ensuring master doesn't break like this.

**Comment by mmlb** [comment]
This should be fixed now after #36
